Resources/Textures/Objects/Misc/brbsign.rsi/meta.json

```
@@ -0,0 +1 @@
+{"version": 1, "size": {"x": 32, "y": 32}, "license": "CC-BY-SA-3.0", "copyright": "Made by efzapa, use or modify it how you please lol.", "states": [{"name": "icon"}]}
```

[Creative Commons Zero](https://creativecommons.org/choose/zero/) might be a better choice than this.

---

Also it says CC-BY-SA-3.0 which [does have certain restrictions](https://creativecommons.org/licenses/by-sa/3.0/). If you really do want to just say "fuck it, use it however, I don't care" pick CC0. Otherwise, maybe look for a more applicable license, because this seems like a copy-paste.

---

Yea, what i originally put in but the sprite was fucky was testing, prob how the old file was formatted so I'll change it

---

```suggestion
{"version": 1, "size": {"x": 32, "y": 32}, "license": "CC0-1.0", "copyright": "Made by efzapa, use or modify it how you please lol.", "states": [{"name": "icon"}]}
```
Makes it CC0 without issues.